api/camunda/camunda-openapi.yaml

```diff
@@ -441,6 +441,58 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
   /user-tasks/{userTaskKey}:
+    get:
+      tags:
+        - User task
```

Comment:
Not sure which one, but either this one or the next endpoint are going to need to change, for consistency. (The next one is tagged with `User Task`, in title case.)

owner: me